cpp/ql/src/experimental/Security/CWE/CWE-193/InvalidPointerDeref.ql

```diff
@@ -154,11 +158,11 @@ predicate pointerAddInstructionHasOperands(
  */
 pragma[nomagic]
 predicate pointerAddInstructionHasBounds(
-  PointerAddInstruction pai, DataFlow::Node sink1, Instruction sink2, int delta
+  PointerAddInstruction pai, DataFlow::Node sink1, DataFlow::Node sink2, int delta
```

Comment:
Why this change? Just a performance thing or is there some semantic difference I'm missing?

Reply:
Oh, right. Sorry. That was part of a solution that I ended up discarding. But I kept it in the PR since I liked the symmetry of having both sinks be nodes instead of one being passed as a node and another being passed as an instruction.

So no, there is no behavior change from this at all.